thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -249,4 +268,44 @@ public void run() {
 
     return Response.ok().build();
   }
+
+
+  @POST
+  @Path("/autotune/{id}")
+  public Response tuneAlertFilter(@PathParam("id") String id,
+      @QueryParam("startTime") Long startTime, @QueryParam("endTime") Long endTime)
```

Comment:
query parameter should include ('alertFilterName') since it may differ from the existing one, you could make the existing one as default.